common/src/main/java/com/revenuecat/purchases/common/AppConfig.kt

```diff
@@ -17,6 +17,7 @@ class AppConfig(
 
     val languageTag: String = context.getLocale()?.toBCP47() ?: ""
     val versionName: String = context.versionName ?: ""
+    val packageName: String = context.packageName
```

Comment:
I couldn't find any character limits for the package name in the android documentation, so not sure what's the limit... I thought about adding a char limit to the header and truncate afterwards but looks like we aren't for other fields like the `versionName` so I decided not to. Lmk if you want to be extra safe and I can add it quickly.

Reply:
do we want to add packageName to the custom equals method?

Reply:
Great catch! Added here #620